packages/@aws-cdk/aws-stepfunctions-tasks/test/databrew/integ.start-job-run.ts

```diff
@@ -0,0 +1,30 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { DataBrewStartJobRun } from '../../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn> : should return execution arn
+ * * aws stepfunctions describe-execution --execution-arn <exection-arn generated before> : should return status as SUCCEEDED and a query-execution-id
```

Comment:
will this actually run successfully? I don't see the data brew job being created anywhere in the test